### ISSUE TITLE
Feature: Extra Examples (outside a simple the viewer)?

### ISSUE THREAD

**Issue by rohit-kumar-j** [rohanpsingh/mujoco-python-viewer#4]
I'm currently working with masse, torques, etc, seeing [this](https://www.youtube.com/playlist?list=PLc7bpbeTIk758Ad3fkSywdxHWpBh9PM0G) series. And was hoping to take the examples directory a bit further (Although I'm not sure how much of this is practical). By creating sort of a tutorial/example with a simple pendulum to obtain units of torque, tuning values of kp, kd, ki etc...
A graphing of PID error like the profiler/sensor section of the [simulate](https://github.com/deepmind/mujoco/blob/87539dbd24825249f6215fb43050d4dd2c9a8fc2/sample/simulate.cc#L255) viewer which generates live graphs.

Perhaps a wiki with these:
- [x] 4-bar-mechanism example, for creation of constraints
[4-bar-example](https://github.com/rohit-kumar-j/mujoco-python-viewer/blob/extended-example-tutorial/examples/simple_linkage.py)
- [x] Simplified [mujoco_viewer.py](https://github.com/rohit-kumar-j/mujoco-python-viewer/tree/extended-example-tutorial/mujoco_viewer) in [extended-example-tutorial](https://github.com/rohit-kumar-j/mujoco-python-viewer/tree/extended-example-tutorial) commits: [`97783ab`](https://github.com/rohit-kumar-j/mujoco-python-viewer/commit/97783ab80e83b041062be6fa84c5fa7baddf93e5) [`5839301`](https://github.com/rohit-kumar-j/mujoco-python-viewer/commit/5839301711c4c16c36b062bb3826631ea0d7c8a9) [`aab2d48`](https://github.com/rohit-kumar-j/mujoco-python-viewer/commit/aab2d48df82cd6cf30a022460d3778d3cf8b00c2)
- [x] Add Sensor Graphs : [Preview](https://github.com/rohanpsingh/mujoco-python-viewer/pull/5#issuecomment-1141172004)
- [x] ~PID tuning and example in wiki~ [mujUI elements are not supported yet in python](https://github.com/deepmind/mujoco/issues/370)

Existing graphing:
![profiler_mujoco](https://user-images.githubusercontent.com/37873142/168340348-cb0df9d1-f855-4887-8497-5563e5bf1de0.gif)

**Comment by rohanpsingh** [comment]
>  Add Sensor Graphs

I understand that you want to add more features in the current viewer, but please note that there is an ongoing effort in the official mujoco github to create python bindings for simulate.cc: https://github.com/deepmind/mujoco/pull/201
Once that is merged, there would most probably be no need for mujoco-python-viewer :)

**Comment by rohit-kumar-j** [comment]
I did not see that. It would be nice to have the simulate viewer in python that is officially supported. Nevertheless, I intended to create a slightly different viewer than simulate(similar to gazebo's [pid tuning](https://vimeo.com/246182184)) and `mujoco-python-viewer` gave a good starting point.

**Comment by rohit-kumar-j** [comment]
Added graph rendering for viewing, although [UI elements for python](https://github.com/deepmind/mujoco/issues/370) are not supported in the mujoco yet #23
